### PR TITLE
Code cleaning

### PR DESCRIPTION
**Pull request by delhomer**
Some cleaning operations to prepare the `0.4` release:
- version updatting in `setup.py`
- some useless modules deletion, some module renaming
- code reading (**in processing**)
- update `README.md`

Fixes the issue #63 .

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
             version = line.strip().split('=')[1].strip(' \'"')
             break
     else:
-        version = '0.3.2'
+        version = '0.4'
```
Comment (garaud) [inline setup.py]: maybe write also the version number in the `__init__.py` file. What to you think?

File: README.md [hunk 2]
```diff
@@ -33,170 +28,167 @@ The picture below shows an example of image generated in this way:
...
-following program calls are supposed to be made from the `source` repository.
++ cv2
++ keras
++ h5py
++ logging
++ matplotlib
++ numpy
++ pandas
++ PIL
++ tensorflow
```
Comment (garaud) [inline README.md:63]: maybe mention the setup.py file with the dependencies and the requitements dev file if needed.

**Review by garaud — CHANGES_REQUESTED** [review]
minor comments

File: setup.py (hunk 1 above)
Comment (delhomer) [reply setup.py]: Yes you're write, it seems to be a good practice. Should I modify the `setup.py` too, or should we keep the version in both scripts?

Comment (garaud) [reply setup.py]: we can keep the version in both scripts. I'll look for another open source projects how they deal with that.

**Comment by delhomer** [comment]
Some new commits have been added to fit your remarks. If everything is OK, we can merge, and release the new code version.

**Review by garaud — APPROVED** [review]

**Pull request merged**